articles/api-auth/references/dashboard-reference.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
Do we want to remove this for now?

Reply:
Yes, glad you removed this. Not sure what it is doing in this branch; I think it's something Tami was working on in another branch, so it shouldn't even be in here. But I will double-check.